src/preferences/dialog/dlgprefcontrols.cpp

```diff
@@ -298,6 +298,46 @@ DlgPrefControls::DlgPrefControls(QWidget * parent, MixxxMainWindow * mixxx,
 
     slotUpdateSchemes();
 
+    double scaleFactor = m_pConfig->getValue(
+            ConfigKey("[Config]", "ScaleFactor"), 1.0);
+
+    //: Entry of the HiDPI scale combo box. %1 is the scale factor-
+    comboBoxScaleFactor->addItem(QString(tr("x %1")).arg(0.5), 0.5);
```

Comment:
A scale factor is usable, using percent would be more consistent with the existing options.
E.g. the waveform zoom level is given in percent.

A tooltip would be a nice touch (``Change the size of text, buttons, and other items.``), as well as for the Auto-Scaling checkbox.

Nit: Moving the new preference option below the Locale option, so the three expanding comboboxes are layered, makes it more easy on the eyes IMO.

RE: the combobox scaling issue, anything the skin designer can do with the combobox styling to allow proper scaling?